apps/ledger-live-desktop/tests/models/Transaction.ts

```diff
@@ -10,3 +10,13 @@ export class Transaction {
     public memoTag?: string,
   ) {}
 }
+
+export class NFTTransaction {
```

Comment:
why don't you inherit already existing object Transaction and add missing properties?